src/store/main-store/MainActions.ts

```diff
@@ -20,4 +20,8 @@ export default class MainActions extends Actions<
     public changeLanguajeToEnglish(): void {
```

Comment:
@gonzalo-plaza please, can you fix this typo "Language", Please don`t forget all the files that contains this typo